addons/fortify/functions/fnc_createObjectMarker.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+
+/*
+ * Author: Seb
+ * Creates a map marker for a created static object but only for sides friendly to the creator side
+ *
+ * Arguments:
+ * 0: Created fortify object <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * _object call acex_fortify_fnc_createObjectMarker
+ *
+ * Public: Yes
```

Comment:
This function doesn't do enough input checking to be considered as public API in my opinion.

Reply:
Apologies, I thought that Public meant a Global result.

Reply:
No worries, that's why there are reviews. :)